dev/NumberBox/TestUI/NumberBoxAxeTestPage.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<local:TestPage
+    x:Class="MUXControlsTestApp.NumberBoxAxeTestPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
+        <!-- These ALL fail our axe scanning so we comment them out. -->
+        <!--
+            <muxc:NumberBox Header="Sample NumberBox"/>
+            <muxc:NumberBox AutomationProperties.Name="Sample NumberBox"/>
+            <muxc:NumberBox Header="Sample NumberBox" SpinButtonPlacementMode="Inline"/>
+            <muxc:NumberBox Header="Sample NumberBox" SpinButtonPlacementMode="Compact"/>
+        -->
```

Comment:
NumberBox doesn't forward header or AutomationProperties.Name to the internal textbox, resulting in a missing UIA name for the internl "edit". @YuliKl @teaP FYI

Reply:
I think this already has an issue... #3505

---
In reply to: [536034266](https://github.com/microsoft/microsoft-ui-xaml/pull/3764#discussion_r536034266) [](ancestors = 536034266)

Reply:
Oh right, forgot about that issue. I'll remove the code then.